pkg/roachprod/vm/aws/aws.go

```diff
@@ -453,6 +456,12 @@ func (o *ProviderOpts) ConfigureClusterFlags(flags *pflag.FlagSet, _ vm.Multiple
 		"Path to json for aws configuration, defaults to predefined configuration")
 }
 
+// ConfigureClusterCleanupFlags implements ProviderOpts.
+func (o *ProviderOpts) ConfigureClusterCleanupFlags(flags *pflag.FlagSet) {
```

Comment:
Don't want to put it in ConfigureClusterFlags because when user will run roachprod command with -h flag for destroyCmd, extendCmd, listCmd, syncCmd, gcCmd, setupSSHCmd, startCmd, pgurlCmd, adminurlCmd
```shell
roachprod syncCmd -h
```
The output will show a flag `--aws-account-ids` which would be misleading.